glusterd2/commands/volumes/commands.go

```diff
@@ -32,6 +32,18 @@ func (c *Command) Routes() route.Routes {
 			HandlerFunc:  volumeExpandHandler},
 		// TODO: Implmement volume reset as
 		// DELETE /volumes/{volname}/options
+		route.Route{
+			Name:        "VolumeOptionGet",
+			Method:      "GET",
+			Pattern:     "/volumes/{volname}/options/{optname}",
+			Version:     1,
+			HandlerFunc: volumeOptionGetHandler},
```

Comment:
Handlers can be merged. So that all the pre validations can remain same. 

   optname, ok := mux.Vars(r)["optname"]
   if ok => one option, else all options